daml-lf/kvutils/BUILD.bazel

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
Can you name this module as `transaction-support` or `transaction-utils`, please? I.e., I would like to avoid tying its name to kvutils as it may be confusing.

Reply:
But it is tied to kvutils? Dropping fetch nodes is not valid in general.

Reply:
Yes, it is tied to `kvutils`, however, that doesn't mean we need to repeat that name in a Maven module's name.

Reply:
let's go for `kv-transaction-support`